config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/config/loader/EntityLoader.java

```diff
@@ -26,6 +27,14 @@
      */
     Object loadSingle(String name, File entitiesFile);
 
+    /**
+     * Load a entities from the given config file.
```

Comment:
You should specify here that this method is currently implemented only for Environment Entities.

Reply:
Done